### PR TITLE
fix: get current dir error

### LINKED ISSUE #2281: make api-test fail
### Issue description

exec "make api-test" failed

### Expected behavior

succes

### How to Reproduce

in apisix-dashboard and make api-test

### Screenshots

image
image

### Environment

- apisix version (cmd: `apisix version`):
- OS (cmd: `uname -a`):Darwin 
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `etcd --version`):
- apisix-dashboard version, if have:
- Browser version, if have:

### Additional context

position https://github.com/apache/apisix-dashboard/blob/master/api/internal/handler/data_loader/route_import_test.go#L92

i think use os.Getwd() instead

i am happy to solve it

### PR DESCRIPTION
**Pull request by jwrookie**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.

**Related issues**

fix/resolve #2281

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: api/internal/handler/data_loader/route_import_test.go [hunk 1]
```diff
@@ -23,7 +23,8 @@ import (
 	"io/ioutil"
 	"mime/multipart"
 	"net/http"
-	"os/exec"
+	"os"
+	"path"
```
Comment (zaunist) [inline api/internal/handler/data_loader/route_import_test.go:27]: Do we need to consider separate path for Windows? Ref  https://pkg.go.dev/path#pkg-overview, the `path` package does not support Windows path.
```
 This package does not deal with Windows paths with drive letters or backslashes;
```

Comment (jwrookie) [reply api/internal/handler/data_loader/route_import_test.go]: sorry, use filepath instead, thks for your check

**Comment by zaunist** [comment]
cc @bzp2010 @starsz @nic-chen

**Review by starsz — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

File: api/internal/handler/data_loader/route_import_test.go (hunk 1 above)
Comment (jwrookie) [reply api/internal/handler/data_loader/route_import_test.go]: @zaunist backend-e2e-test error,  [click](https://github.com/apache/apisix-dashboard/runs/4756297493?check_suite_focus=true)

can you check it and rerun? thks

**Comment by zaunist** [comment]
cc @bzp2010

**Review by 1502shivam-singh — APPROVED** [review]

**Review by zaunist — APPROVED** [review]

**Pull request merged**